BlueprintUI/Sources/Layout/Builder/Builder.swift

```diff
@@ -0,0 +1,10 @@
+/// Generic result builder for converting blocks of `Child...` into `[Child]`.
+@_functionBuilder // change to @resultBuilder when we upgrade to Swift 5.4.
+public struct Builder<Child> {
```

Comment:
It would be nice to include the other builders here too – eg buildEither for ifs, etc..

Reply:
I'm not exactly sure what you mean here. Could you give an example of what this would look like when used? Can't you achieve the same syntax for an `if` with normal closures?